person_capture/gui_app.py

```
@@ -634,6 +634,46 @@ def _jsonable_cfg_value(v):
+            safe = str(fallback or "video")
+        if len(safe) > max_len:
+            safe = safe[:max_len].rstrip("._-") or str(fallback or "video")
+        return safe
+
+    @staticmethod
+    def _source_filename_prefix(path: str) -> str:
+        src = str(path or "").strip()
+        stem = Processor._safe_filename_component(Path(src).stem if src else "", fallback="video")
+        try:
+            ident_path = os.path.abspath(src) if src else ""
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)